package.json

```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "amqplib": "^0.5.1",
+    "bluebird": "^3.5.0",
```

Comment:
@krisread how come this moved to dependencies from dev-dependencies?

Reply:
I used it in a former version of this PR then forgot to remove it as I guess that line was refactored out